sound/soc/sof/ipc4-topology.h

```diff
@@ -349,6 +349,12 @@ struct sof_ipc4_process {
 	void *ipc_config_data;
 	uint32_t ipc_config_size;
 	struct sof_ipc4_msg msg;
+	/*
+	 * init_blob_has_out_fmt is used to describe whether there is output format
+	 * (struct sof_ipc4_audio_format output_format) in the module init instance
+	 * ipc message payload blob.
+	 */
+	bool init_blob_has_out_fmt;
```

Comment:
this flag only applies to the init message? Is there other message regarding to the module which is not init?

Reply:
yes, this flag is only for init message. there are other messages for the module(set_config, set_large_config), but none of them touch this flag.

Reply:
Right, and if the blob have "source format"? ;)
What was wrong with the original "payload_with_output_fmt" name of the flag?

Reply:
I agree we should keep the same flag name too.

Reply:
> Right, and if the blob have "source format"? ;) What was wrong with the original "payload_with_output_fmt" name of the flag?

Nothing wrong here, I just want to be more precise. even `payload_with_output_fmt`, it really means the init_payload_with_output_fmt, just as I said, this token is only used with the module init ipc message payload, not elsewhere.

@ranj063 @ujfalusi If you insist, I can keep the name `payload_with_output_fmt`,

Reply:
please shorten the token name as needed. As long as it matches the description, it should be fine